closure/goog/net/jsloader.js

```diff
@@ -119,8 +119,9 @@ goog.net.jsloader.loadMany = function(uris, opt_options) {
     if (uris.length) {
       deferred.addBoth(popAndLoadNextScript);
     }
+    return deferred;
   };
-  popAndLoadNextScript();
+  return popAndLoadNextScript();
```

Comment:
ok will do in a week or so.

On 30 Jul 2015, at 19:56, Joel Martinez <notifications@github.com> wrote:

> In closure/goog/net/jsloader.js:
> 
> >    };
> > -  popAndLoadNextScript();
> > +  return popAndLoadNextScript();
> Add a @return type to loadMany.
> 
> —
> Reply to this email directly or view it on GitHub.
>